pkg/stores/partition/store.go

```diff
@@ -124,7 +124,7 @@ func (s *Store) ByID(apiOp *types.APIRequest, schema *types.APISchema, id string
 	if err != nil {
 		return types.APIObject{}, err
 	}
-	return toAPI(schema, obj, warnings), nil
+	return toAPI(schema, moveToUnderscore(obj), warnings), nil
```

Comment:
If this looks up a single object, do we need to do `moveToUnderscore`? I noticed we don't for `List` and `Watch`